.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
+labels: bug
+---
+
+<!-- Lines like this are comments and will be invisible -->
```

Comment:
You need to add this line in every template.

Reply:
I don't understand why you deem this necessary. As the comments are invisible, users *don't* need to remove them: That was the idea - instead of having lots of text in the issue template that people need to delete (or it clutters the issue if they don't), just some helpfully guiding comments. Also, I don't think users should face all-caps when they're already kind enough to give us some feedback in issues. I think we should make it as pleasant as possible to leave us an issue - at least to me, the proposed change sounds like we don't have a particularly high opinion of our users - definitely not something I'd like to read if I am about to give a project feedback.

Reply:
> Also, I don't think users should face all-caps when they're already kind enough to give us some feedback in issues.

More text - less likely people read it, that is proven by using other templates and in many other projects. Not everything in the template is within HTML comments, and that text should be deleted. For me, the capitalized text is not insulting, it just a way of bringing attention to important bits in a text where you cannot use colour or more advanced approaches.

Why capitalized message can be important:

a. some people may not know what `<!-- -->` means, and we need to explain it.
b. some text requires removing and we need to shout about it.
c. all points above somehow should be emphasized in the text, so that a user reads it first.

Reply:
I agree with ti here. Even if the intention's positive, it doesn't come across that way

Reply:
Capitalized text is normal and used broadly. See another example: https://raw.githubusercontent.com/stevemao/github-issue-templates/master/bugs-only/ISSUE_TEMPLATE.md.

Reply:
At a): that's what this line already says. At b): what lines do you think need removing? The point of putting notes in comments was that they *don't* need removing... At c): I would prefer the risk of someone not noticing it (and perhaps leaving a sub-optimal issue report) over the risk of putting someone off from reporting an issue in the first place.
Someone else doing it that way doesn't necessarily make it right, or the best, way.

Reply:
```suggestion
<!-- Lines like this, enclosed in <!-- -->, are comments and will be invisible -->
```

Reply:
@awav I guess there's a subtlety to it, but as I say, what matters is how it reads not how we want it to read. There are cases where caps is OK, and I think the example you gave is OK, but I think you have to be careful what words you emphasise (by caps or punctuation) because of how it changes the tone